tracing/src/main/java/com/palantir/tracing/Tracers.java

```diff
@@ -281,21 +290,25 @@ public static Runnable wrapWithAlternateTraceId(String traceId, Runnable delegat
         return wrapWithAlternateTraceId(traceId, DEFAULT_ROOT_SPAN_OPERATION, delegate);
     }
 
+    public static Runnable wrapWithAlternateTraceId(String traceId, String operation, Runnable delegate) {
```

Comment:
mark this and the other compatibility methods as deprecated?

Reply:
done